packages/chevrotain/src/parse/grammar/gast/gast_public.ts

```diff
@@ -290,11 +299,17 @@ export function serializeProduction(node: IProduction): ISerializedGast {
   }
   /* istanbul ignore else */
   if (node instanceof NonTerminal) {
-    return <ISerializedNonTerminal>{
+    const serializedNonTerminal: ISerializedNonTerminal = {
       type: "NonTerminal",
       name: node.nonTerminalName,
       idx: node.idx
     }
+
+    if (node.label) {
```

Comment:
should we check for hasOwnProperty instead?
The utils pkg may have an "has" utility

Reply:
We could, but would require that the label property only exists when it has a string value.  Which currently isn't the case.  i.e. when constructing a NonTerminal the options is initialized with `label: options?.LABEL`.

I've changed the check to `isString(node.label)` instead